### DISCUSSION TITLE
Configuring values for options

### DISCUSSION THREAD

**Discussion by missmatsuko** [solspace/craft-express-forms#62] [Feature Requests]
### Description
I don't see any way to configure values for an options field type in the CMS.

So, it seems that content authors wouldn't be able to make dropdowns/radio groups/checkbox groups with custom values.

Is that intentional or am I just missing something? 

If there were a way to configure values and a way to select either type (dropdown/radio/checkbox) or behaviour (select 1 or select multi), then I would be able to use this plugin.

**Comment by kjmartens** [comment]
We appreciate your feedback on this. I can understand how a feature like this would be handy for Express Forms, but our intended purpose for it (and to differentiate itself from Freeform) is to be simpler and geared towards users that enjoy manually building their forms directly inside their templates. We know that Express Forms isn't for everyone. :)

**Comment by missmatsuko** [comment]
I do like the form builder experience in Express Forms more than Solspace. It's simpler and more intuitive (especially with the advanced UI turned off, so it behaves more like Matrix used elsewhere in the CMS). But I don't think we could use it on any client projects if content authors can't create forms with select/radio groups/checkbox groups.

**Comment by kjmartens** [comment]
For what it's worth, we do have many more form builder improvements in the works for Freeform. :)

**Comment by outline4** [comment]
I would consider using express forms if I could make dropdowns/radio groups/checkbox groups with custom values. It's the only thing that is holding me from using it. 

I mean, you can also specify stuff for uploads, why not for checkboxes etc?  

If it's a political choice, then why bother making a second form tool? 

I would prefer express forms over freeform. because over its simplicity.

**Comment by kjmartens** [comment]
@outline4 File Uploads absolutely require some level of control to be configured in the back end, checkbox groups, select fields, etc do not necessarily, and can be simply hardcoded in the template. Based on some feedback about Freeform, some customers hate that they have to predefine option values inside the CP for these kinds of fields, and we're aware of so many customers that prefer to hardcode their templates as much as possible.

The idea of Express Forms was to assume as little information as possible and also be a much more robust free offering than the native Craft Contact Form, which currently requires you to install 5 other 3rd party helper plugins before it begins to come close to what Express Forms offers. At the same time, we understand that there will be some customers inbetween, where they like most of the simplicity of Express Forms, but it's too simple for their needs in some regards, while Freeform may have too many features or assumptions on how the forms and data should be rendered and handled, etc.

As mentioned earlier, we are continually making improvements to Freeform based on the feedback we receive, and have a very large update coming to Freeform that includes a taste of Express Forms that many users will appreciate. At the same time, we're considering possibly coming up with a way to allow users to optionally predefine field options for Express Forms in the future. :)

**Comment by outline4** [comment]
>  At the same time, we're considering possibly coming up with a way to allow users to optionally predefine field options for Express Forms in the future. :)

That's all I wanted to hear ;)  
Thanks for listening!

**Comment by medoingthings** [comment]
+1 Being able to create options would be highly valuable. And Freeform would still have enough differentiation from Express Forms. If it would be only available in the Pro version, that would be fine, too. I don’t need all the bells and whistles from Freeform, I just need a set of options :)
